File: testresources/csharp/benchmark/Domain/Direct/Violating/DeclarationParameter_GenericType_OneTypeParameter.cs
using System.Collections.Generic;
using Technology.Direct.Dao;

namespace Domain.Direct.Violating
{
	public class DeclarationParameter_GenericType_OneTypeParameter
	{
		public String testUsageOfInstanceVariableDefinedByGenericTypeOneParameterType(List<ProfileDAO> pDao)
		{
			return pDao.get(0).toString();
		}
	}
}

File: testresources/java/accuracy/src/domain/direct/violating/DeclarationParameter_GenericType_OneTypeParameter.java
package domain.direct.violating;

import java.util.ArrayList;
import technology.direct.dao.ProfileDAO;

public class DeclarationParameter_GenericType_OneTypeParameter {
	
	public String testUsageOfInstanceVariableDefinedByGenericTypeOneParameterType(ArrayList<ProfileDAO> pDao){
		return pDao.get(0).toString();
	}
}